spec/lib/facilities/ppms/client_spec.rb

```diff
@@ -1,89 +1,183 @@
+        r = Facilities::PPMS::Client.new.pos_locator(params, '17')
+        expect(r.length).to be 5
+        expect(r[0]).to have_attributes(
+          ProviderIdentifier: '1629245311',
+          Name: 'MinuteClinic LLC',
+          AddressStreet: '2010 S Dobson Rd',
+          AddressCity: 'Chandler',
+          AddressStateProvince: 'AZ',
+          AddressPostalCode: '85286',
+          Email: nil,
+          MainPhone: nil,
+          CareSitePhoneNumber: '8663892727',
+          OrganizationFax: nil,
+          ContactMethod: nil,
+          IsAcceptingNewPatients: 'false',
+          ProviderGender: 'NotSpecified',
+          ProviderSpecialties: [],
+          Latitude: 33.275526,
+          Longitude: -111.877057,
+          Miles: 0.79
+        )
       end
+    end
+  end
 
-      it 'returns a CareSite' do
-        VCR.use_cassette('facilities/va/ppms', match_requests_on: [regex_matcher]) do
-          r = PPMS::Client.new.provider_caresites(1_427_435_759)
-          expect(r.length).to be > 0
-          expect(r[0]['Longitude']).not_to be(nil)
-        end
+  describe '#provider_info' do
+    it 'gets additional attributes for the provider' do
+      VCR.use_cassette('facilities/va/ppms', match_requests_on: %i[path query]) do
+        r = Facilities::PPMS::Client.new.provider_info(1_407_842_941)
+        expect(r).to have_attributes(
+          AddressCity: nil,
+          AddressPostalCode: nil,
+          AddressStateProvince: nil,
+          AddressStreet: nil,
+          CareSitePhoneNumber: nil,
+          ContactMethod: nil,
+          Email: 'evfa1@hotmail.com',
+          IsAcceptingNewPatients: 'true',
+          Latitude: nil,
+          Longitude: nil,
+          MainPhone: '4809241552',
+          Miles: nil,
+          Name: 'Freed, Lewis ',
+          OrganizationFax: '4809241553',
+          ProviderGender: 'Male',
+          ProviderIdentifier: '1407842941'
+        )
+        expect(r['ProviderSpecialties'].each_with_object(Hash.new(0)) do |specialty, count|
+          count[specialty['CodedSpecialty']] += 1
+        end).to match('213E00000X' => 41)
       end
+    end
+  end
 
-      it 'returns Services' do
-        VCR.use_cassette('facilities/va/ppms', match_requests_on: [regex_matcher]) do
-          r = PPMS::Client.new.provider_services(1_427_435_759)
-          expect(r.length).to be > 0
-          expect(r[0]['Longitude']).not_to be(nil)
-          expect(r[0]['CareSiteAddressStreet']).not_to be(nil)
-        end
+  describe '#provider_services' do
+    it 'returns Services' do
+      VCR.use_cassette('facilities/va/ppms', match_requests_on: %i[path query]) do
+        r = Facilities::PPMS::Client.new.provider_services(1_407_842_941)
+        expect(r.each_with_object(Hash.new { |h, k| h[k] = Hash.new(0) }) do |service, count|
+          %w[Name AffiliationName RelationshipName CareSiteName CareSiteAddressZipCode].each do |key|
+            count[key][service[key]] += 1
+          end
```

Comment:
This might be pretty hard to figure out when someone reads in the future -- though i get the alternative is pretty huge. Would it be worth matching to a fixture so the code is a little simpler?

```
expect(r).to match(JSON.parse(some_giant_fixture_on_disk))
```

Might be just as easy to consider a simpler cassette if that path makes sense.

Reply:
I'm not happy with this one at all. currently, we jump through two objects to [get_body](https://github.com/department-of-veterans-affairs/vets-api/blob/4129-facility-locator-urgent-care-service-type-splits/lib/facilities/ppms/response.rb#L31-L33)

This is something I'm going to change in the future but I'm fine with this test case going away since we don't do anything with the response. it's mostly here so I could get the 100% code coverage.